src/NovelAudioChannel.cpp

```diff
@@ -0,0 +1,82 @@
+//
+// Created by krjoh on 29/08/2019.
+//
+
+#include "NovelAudioChannel.h"
+#include <string>
+#include <map>
+
+namespace NovelRT {
+NovelAudioChannel::NovelAudioChannel(std::string input, bool isBgm, int channel){
+  if (isBgm)
+  {
+    _loadFile(input, 0);
+    _isMusic = true;
+    _channel = 0;
+  }
+  else
+  {
+    _loadFile(input, channel);
+    _isMusic = false;
+    _channel = channel;
+  }
+}
+
+bool NovelAudioChannel::_loadFile(std::string in, int channelAssigned){
+  if (!channelAssigned == 0)
```

Comment:
uhhh that hurts my head to read. Can you clarify this? I'm unsure what the condition is supposed to be.